<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>281</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>521</width>
    <height>281</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>521</width>
    <height>281</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Anime List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_org_name">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Оригинальное название:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Русское название:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="e_rus_name">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="s_series">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер серии:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Статус:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_add">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_change">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_del">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_find">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
